packages/form-field/src/FormFieldInputContainer/FormFieldInputContainer.tsx

```diff
@@ -72,7 +72,7 @@ export const FormFieldInputContainer = forwardRef<
         <div className={iconsWrapperStyles}>
           {state === FormFieldState.Valid && !disabled && (
             <Icon
-              role="presentation"
+              aria-hidden
```

Comment:
out of curiosity - what makes this right?

Reply:
In the icon component, if the `role` is `presentation` then all other attributes are omitted, which makes it hard to query for the error/warning icon. This way I can query for the icon with the `title` attribute.

Reply:
got it!

Reply:
created a [ticket](https://jira.mongodb.org/browse/LG-4149) to investigate